docs/integrations_api.yml

```diff
@@ -236,24 +265,19 @@ paths:
 
   /devices/{id}/attributes:
```

Comment:
This functionality is not implemented yet, let's drop it from doc and move it to the ticket so we have parts of documentation to reuse. https://tracker.mender.io/browse/MEN-638